tests/ttnn/unit_tests/test_expand.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import ttnn
+import pytest
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [1, 32],
+        [8, 1],
+        [32, 1],
+    ],
+    ids=["1d", "2d", "l2d"],
+)
+@pytest.mark.parametrize(
+    "output_shape",
+    [
+        [-1, 32],
+        [32, -1, 32],
+        [32, 32, -1, 32],
+        [32, 32, 32, -1, 32],
+        [4, 4, 4096, -1, 32],
+    ],
+    ids=["2d", "3d", "4d", "5d", "random_large_2d"],
```

Comment:
Last test name should be `random_large_5d`?

Reply:
Oops, yeah it should have been 5d